pkg/virt-operator/resource/apply/certificates.go

```diff
@@ -7,25 +7,43 @@ import (
 )
 
 func getCADuration(config *k8sv1.KubeVirtSelfSignConfiguration) *metav1.Duration {
-	if config == nil || config.CARotateInterval == nil {
-		return &metav1.Duration{Duration: Duration7d}
+	if config != nil {
```

Comment:
nit: we could reduce complexity a bit if we instead define `defaultDuration` and do 

```
if config == nil {
   return defaultDuration
}
...
return defaultDuration
```

we then have to return it twice so not sure :shrug: